### PR TITLE
Make text scale again and also make grid scale

### PR DESCRIPTION
**Pull request by SheepTester**
Hurra! The era under the wrath of the Formidable D. P. R. is over!

### CODE REVIEW

**Comment by anematode** [comment]
A​​​​​l​l​​​​​​​​​​​​​​​​​​​ ​​​​​​​​​​​​​​​​​​​​p​​​​​r​​​​​​​​​​​​​​​​​​ai​​​​​s​​​​​​​e​​​​​​​ G​​​​​​​​​​​​​​​​​​​​a​​​​​​​​m​​​​​​​​​e​​​​​​​​​​​​​​​​​​​pr​​​​​​​​​o​​​​​​​​​​​​​​​​​​​5!​ ​​​​​​​​​​​​​​Al​​​​​l​ ​​​​​​​​​​​​​​​​​​​p​​​​​​​​​​​​​​​​​​​​r​​​​​a​​​​​​​​​​​​​​​​​​is​​​​​e​​​​​​​ ​​​​​​​Ga​​​​​m​e​​​​​​​​​​​​​​​​​​​p​​​​​​​​​​​​​​​​​​​​r​​​​​o​​​​​​​​​​​​​​​​​​5!​​​​​ ​​​​​​​A​​​​​​​ll​​​​​​​​​​​​​​​​​​​​ ​​​​​​​​p​​​​​​​​​r​​​​​​​​​​​​​​​​​​​ai​​​​​​​​​s​​​​​​​​​​​​​​​​​​​e ​G​​​​​​​​​​​​​​am​​​​​e​p​​​​​​​​​​​​​​​​​​​r​​​​​​​​​​​​​​​​​​​​o​​​​​5​​​​​​​​​​​​​​​​​​! ​​​​​A​​​​​​​l​​​​​​​l ​​​​​p​r​​​​​​​​​​​​​​​​​​​a​​​​​​​​​​​​​​​​​​​​i​​​​​s​​​​​​​​​​​​​​​​​​e ​​​​​G​​​​​​​a​​​​​​​me​​​​​​​​​​​​​​​​​​​​p​​​​​​​​r​​​​​​​​​o​​​​​​​​​​​​​​​​​​​5!​​​​​​​​​ ​​​​​​​​​​​​​​​​​​​Al​l​​​​​​​​​​​​​​ p​​​​​r​a​​​​​​​​​​​​​​​​​​​i​​​​​​​​​​​​​​​​​​​​s​​​​​e​​​​​​​​​​​​​​​​​​ G​​​​​a​​​​​​​m​​​​​​​epro5! All praise Gamepro5!

File: src/gridlines.js [hunk 1]
```diff
@@ -522,7 +522,7 @@ class AutoGridlines extends Gridlines {
         }
       }
 
-      let ideal_xy = this.context.pixelToCartesianV(this.normal.ideal_dist / utils.dpr, this.normal.ideal_dist / utils.dpr);
+      let ideal_xy = this.context.pixelToCartesianV(this.normal.ideal_dist, this.normal.ideal_dist);
```
Comment (SheepTester) [inline src/gridlines.js:525]: this change and the next change were suggested by timothy

File: src/gridlines.js [hunk 2]
```diff
@@ -676,7 +676,7 @@ class AutoGridlines extends Gridlines {
             bl: getTextBaseline(labely.align), // baseline
             ta: getTextAlign(labely.align), // textalign
             lpos: labely.location,
-            font: `${labely.font_size / utils.dpr}px ${labely.font}`,
+            font: `${labely.font_size}px ${labely.font}`,
```
Comment (SheepTester) [inline src/gridlines.js:679]: 8/10 fish slightly undercooked, but otherwise enjoyable

**Comment by anematode** [comment]
Hooray!!

**Pull request merged**